mmdet/models/backbones/mobilenetv2.py

```diff
@@ -0,0 +1,253 @@
+import torch
+import torch.nn as nn
+from mmcv.cnn import (constant_init, kaiming_init, normal_init)
+from ..registry import BACKBONES
+
+
+def conv_bn(inp, oup, stride, groups=1, act_fn=nn.ReLU):
```

Comment:
We have a [ConvModule](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/utils/conv_module.py) in this codebase. Use the designed module to act conv/bn/relu would be better.